utils/utils.go

```diff
@@ -47,9 +47,16 @@ func ConvertAPIStringToDate(date string) time.Time {
 	return time.Date(year, time.Month(month), day, 0, 0, 0, 0, time.UTC)
 }
 
+func SendResponseBack(w http.ResponseWriter, body interface{}, httpStatusCode int) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(httpStatusCode)
+	json.NewEncoder(w).Encode(body)
+}
+
 func SendErrorBack(w http.ResponseWriter, err error, info string) {
 	log.Println(info + ":", err)
 	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusBadRequest)
 	var iceCreamError errors_models.IceCreamErrors
```

Comment:
hmm, I think I've seen this same change in a different PR

Reply:
Yeah, will remove this before I merge this PR.